_includes/partials/contributor.html

```diff
@@ -1,62 +1,72 @@
-<div class="contributor flex"> 
-    <div class="container {% if page.full_width %}max-container{% endif %}">
-        <h2>Community Members</h2>
-        <p>A warm and welcoming collection of open sourcers.</p>
-
-        <ul class="contributor">  
-        <!-- List all Contributors -->
-        {% if "community.html" == page.name %}
-            {% for page in site.contributors %}
-                {% if page.layout != 'archive' %}
-                <li>
-                    <div class="square-image">
-                        <a href="{{page.url}}">
-                            <img src="{% include relative-src.html src=page.image_path %}" alt="{{ page.name }}"/>
-                        </a>
-                    </div>
-                    <div class="names">{{ page.name }}</a></div>
-
-                    {% if page.twitter %}
-
-                        <div class="twitter"><img src="{% include relative-src.html src='../images/social/twitter.png'%}"> 
-                            <a target="_blank" href="https://twitter.com/{{ page.twitter }}">{{ page.twitter }}</a></div>
-
-                    {% endif %}
-                    {% if page.github %}
-                        
-                        <div class="github"><img src="{% include relative-src.html src='../images/social/github.png'%}"> 
-                            <a target="_blank" href="https://github.com/{{ page.github }}">{{ page.github }}</a></div>
+<!DOCTYPE html>
```

Comment:
The contributor.html file is intentionally a partial html document. This file should not have <html><head><body> tags in it. Those pages are added elsewhere as the site is built.